app/models/media_resource.rb

```diff
@@ -1,4 +1,7 @@
 class MediaResource < ApplicationRecord
+  AUDIO_EXTENSIONS = %w[aac aiff au flac m4a m4b mp2 mp3 ogg wav]
+  VIDEO_EXTENSIONS = %w[avi flv m4v mov mp4 webm wmv]
+
```

Comment:
Would it be worthwhile to include an `mp4` as an audio extension? Or is it problematic to always consider mp4s a video?